webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/ExpectedReportsJdbcRepository.scala

```diff
@@ -223,8 +224,8 @@ class UpdateExpectedReportsJdbcRepository(
 
   val logger = ReportLogger
 
-  override def closeNodeConfigurations(nodeId: NodeId): Box[NodeId] = {
-    transactRunBox(xa => sql"""
+  override def closeNodeConfigurationsPure(nodeId: NodeId): IOResult[NodeId] = {
```

Comment:
I need to look in other parts of code if this is still ran

Reply:
since it's now a ZIO instead of a box

Reply:
Ok found usaged in RemoveNodeService and they are ok

Reply:
Not sure it should be named  closeNodeConfigurationsPure if there is no closeNodeConfigurations function and that other function 'deleteNodeIndos' which is pure too does not have Pure in its name

So would rename fo closeNodeConfigurations for simpler names

Reply:
no, I can't renamed, it breaks compatibility, as explained in the PR comment (the part about bridge methods)